# Copyright 2017 ZT Prentner IT GmbH
#
# This program is free software: you can redistribute it and/or modify
# it under the terms of the GNU Affero General Public License as published
# by the Free Software Foundation, either version 3 of the License, or
# (at your option) any later version.
#
# This program is distributed in the hope that it will be useful,
# but WITHOUT ANY WARRANTY; without even the implied warranty of
# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
# GNU Affero General Public License for more details.
#
# You should have received a copy of the GNU Affero General Public License
# along with this program.  If not, see <http://www.gnu.org/licenses/>.
#
# German translations for RKTool package.
msgid ""
msgstr ""
"Project-Id-Version: 0.1\n"
"POT-Creation-Date: 2017-02-07 15:22+CET\n"
"PO-Revision-Date: 2016-05-30 23:05+0200\n"
"Last-Translator: Mino Sharkhawy\n"
"Language-Team: German\n"
"Language: de\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Generated-By: pygettext.py 1.5\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"

#: cashreg.py:98
msgid "Invalid turnover counter size."
msgstr "Umsatzzähler-Länge ungültig."

#: cashreg.py:137 main.py:364 receipt.py:86 rktool.py:364
msgid "Invalid key."
msgstr "Ungültiger Schlüssel."

#: demo.py:82
msgid "The number of receipts must be at least 1."
msgstr "Die Anzahl der Rechnungen muss mindestens 1 sein."

#: main.py:164 rktool.py:164
msgid "Error"
msgstr "Fehler"

#: main.py:231 main.py:306 rktool.py:231 rktool.py:306
msgid "Valid Signature"
msgstr "Gültige Signatur"

#: main.py:236 rktool.py:236
msgid "No Signature"
msgstr "Keine Signatur"

#: main.py:268 rktool.py:268
msgid "Signature system broken"
msgstr "Sicherheitseinrichtung ausgefallen"

#: main.py:270 rktool.py:270
msgid "ZDA ID"
msgstr "ZDA ID"

#: main.py:271 rktool.py:271
msgid "Cash Register ID"
msgstr "Kassen-ID"

#: main.py:272 rktool.py:272
msgid "Receipt ID"
msgstr "Belegnummer"

#: main.py:273 rktool.py:273
msgid "Timestamp"
msgstr "Datum & Uhrzeit"

#: main.py:274 rktool.py:274
msgid "Sum Tax Normal"
msgstr "Betrag Satz Normal"

#: main.py:275 rktool.py:275
msgid "Sum Tax Reduced 1"
msgstr "Betrag Satz Ermäßigt 1"

#: main.py:276 rktool.py:276
msgid "Sum Tax Reduced 2"
msgstr "Betrag Satz Ermäßigt 2"

#: main.py:277 rktool.py:277
msgid "Sum Tax Zero"
msgstr "Betrag Satz Null"

#: main.py:278 rktool.py:278
msgid "Sum Tax Special"
msgstr "Betrag Satz Besonders"

#: main.py:279 rktool.py:279
msgid "Turnover Counter"
msgstr "Umsatzzähler"

#: main.py:280 rktool.py:280
msgid "Certificate Serial/Key ID"
msgstr "Zertifikat-Seriennummer/Schlüssel ID"

#: main.py:281 rktool.py:281
msgid "Chaining Value"
msgstr "Verkettungswert"

#: main.py:282 rktool.py:282
msgid "Signature"
msgstr "Signatur"

#: main.py:287 main.py:629 rktool.py:287 rktool.py:629
msgid "Verifying..."
msgstr "Überprüfe..."

#: main.py:300 main.py:610 main.py:645 rktool.kv:145 rktool.kv:248
#: rktool.py:300 rktool.py:610 rktool.py:645
msgid "Verify"
msgstr "Überprüfen"

#: main.py:327 main.py:698 rktool.kv:233 rktool.py:327 rktool.py:698
msgid "Load AES Key"
msgstr "AES Schlüssel laden"

#: main.py:348 main.py:718 main.py:867 rktool.py:348 rktool.py:718
#: rktool.py:867
msgid "Malformed crypto container"
msgstr "Ungültiges Schlüsselmaterial"

#: main.py:409 rktool.py:409
msgid "No image taken."
msgstr "Kein Bild aufgenommen."

#: main.py:418 rktool.py:418
msgid "Failed to compress image."
msgstr "Fehler beim Komprimieren des Bildes."

#: main.py:437 main.py:458 rktool.py:437 rktool.py:458
msgid "No QR codes found."
msgstr "Keine QR-Codes gefunden."

#: main.py:445 rktool.py:445
msgid "Load Receipt"
msgstr "Rechnung laden"

#: main.py:558 rktool.py:558
msgid "Group %d"
msgstr "Gruppe %d"

#: main.py:570 main.py:577 main.py:761 rktool.py:570 rktool.py:577
#: rktool.py:761
msgid "Serial: "
msgstr "Seriennummer: "

#: main.py:652 rktool.py:652
msgid "Valid DEP"
msgstr "Gültiges DEP"

#: main.py:670 rktool.kv:230 rktool.py:670
msgid "Load DEP"
msgstr "DEP laden"

#: main.py:729 rktool.py:729
msgid "Public Keys"
msgstr "Öffentliche Schlüssel"

#: main.py:731 rktool.py:731
msgid "Certificates"
msgstr "Zertifikate"

#: main.py:766 rktool.py:766
msgid "Key ID: "
msgstr "Schlüssel ID: "

#: main.py:779 rktool.py:779
msgid "Load PEM Public Key"
msgstr "Öffentlichen Schlüssel (PEM) laden"

#: main.py:784 rktool.py:784
msgid "Load PEM Certificate"
msgstr "Zertifikat (PEM) laden "

#: main.py:805 rktool.py:805
msgid "Enter Public Key ID"
msgstr "ID des öffentlichen Schlüssels eingeben"

#: main.py:835 rktool.py:835
msgid "Load Key Store"
msgstr "Schlüsselspeicher laden"

#: main.py:841 rktool.py:841
msgid "Save Key Store"
msgstr "Schlüsselspeicher speichern"

#: receipt.py:42 verify.py:90
msgid "At receipt \"{0}\": {1}"
msgstr "Bei Rechnung \"{0}\": {1}"

#: receipt.py:61
msgid "Malformed receipt."
msgstr "Ungültige Rechnung."

#: receipt.py:70
msgid "Unknown algorithm."
msgstr "Unbekannter Algorithmus."

#: receipt.py:78
msgid "Algorithm mismatch."
msgstr "Inkompatible Algorithmen."

#: receipt.py:115
msgid "Unknown Receipt"
msgstr "Unbekannte Rechnung."

#: receipt.py:244 receipt.py:358 receipt.py:413 receipt.py:509
msgid "You need to sign the receipt first."
msgstr "Die Rechnung muss zuerst signiert werden."

#: receipt.py:549
msgid "Can't decrypt turnover counter, this is a dummy receipt."
msgstr "Kann Umsatzzähler nicht entschlüsseln. Dies ist ein Trainingsbeleg."

#: receipt.py:551
msgid "Can't decrypt turnover counter, this is a reversal receipt."
msgstr "Kann Umsatzzähler nicht entschlüsseln. Dies ist ein Stornobeleg."

#: receipt.py:587 verify_receipt.py:403
msgid "Input format must be one of %s."
msgstr "Eingabeformat muss eines aus %s sein."

#: receipt.py:591
msgid "Output format must be one of %s."
msgstr "Ausgabeformat muss eines aus %s sein."

#: rktool.kv:52 rktool.kv:75 rktool.kv:93
msgid "Cancel"
msgstr "Abbrechen"

#: rktool.kv:55
msgid "Load"
msgstr "Laden"

#: rktool.kv:78
msgid "Save"
msgstr "Speichern"

#: rktool.kv:96
msgid "Ok"
msgstr "Ok"

#: rktool.kv:128
msgid "Receipt"
msgstr "Rechnung"

#: rktool.kv:132 rktool.kv:286
msgid "Exit"
msgstr "Beenden"

#: rktool.kv:149
msgid "Decrypt Turnover Counter"
msgstr "Umsatzzähler entschlüsseln"

#: rktool.kv:197
msgid "JWS string"
msgstr "JWS-Zeichenkette"

#: rktool.kv:203
msgid "QR code"
msgstr "QR-Code"

#: rktool.kv:208
msgid "OCR code"
msgstr "OCR-Code"

#: rktool.kv:213
msgid "URL"
msgstr "URL"

#: rktool.kv:217
msgid "Read Receipt"
msgstr "Rechnung lesen"

#: rktool.kv:268
msgid "Import"
msgstr "Importieren"

#: rktool.kv:271
msgid "Export"
msgstr "Exportieren"

#: rktool.kv:279
msgid "Registrierkassen Tool"
msgstr "Registrierkassen Tool"

#: rktool.kv:297
msgid "Verify Receipt"
msgstr "Rechnung überprüfen"

#: rktool.kv:301
msgid "Verify DEP"
msgstr "DEP überprüfen"

#: rktool.kv:305
msgid "Key Store"
msgstr "Schlüsselspeicher"

#: run_test.py:63
msgid "Need key material for {} signature devices, got {} key pairs."
msgstr ""
"Brauche Schlüsselmaterial für {} Sicherheitseinrichtungen, habe {} "
"Schlüsselpaare bekommen."

#: run_test.py:269
msgid "I need keys and certificates for %d signature devices."
msgstr "Ich brauche Schlüssel und Zertifikate für %d Sicherheitseinrichtungen."

#: run_test.py:281 test_verify.py:245
msgid "Turnover counter size needs to be between 5 and 16."
msgstr "Umsatzzähler-Länge muss zwischen 5 und 16 sein."

#: sigsys.py:153
msgid "Selected algorithm not supported."
msgstr "Gewählter Algorithmus wird nicht unterstützt."

#: test_verify.py:69 test_verify.py:72 test_verify.py:77
msgid "no Exception"
msgstr "kein Fehler"

#: test_verify.py:115
msgid "Expected {} in turnover counter but got {}."
msgstr "Erwartete {} im Umsatzzähler, aber habe {} bekommen."

#: test_verify.py:128
msgid "Expected \"{}\" but got \"{}\", message: \"{}\""
msgstr "Erwartete \"{}\", aber habe \"{}\" bekommen, Nachricht: \"{}\""

#: test_verify.py:135
msgid "Expected \"{}\" at receipt \"{}\" but it occured at \"{}\" instead"
msgstr ""
"Erwartete \"{}\" bei Rechnung \"{}\", aber habe es stattdessen bei \"{}\" "
"bekommen"

#: test_verify.py:171
msgid "Result mismatch: without parsing {}:>{}<, with parsing {}:>{}<"
msgstr "Ergebnisse verschieden: ohne parsing {}:>{}<, mit parsing {}:>{}<"

#: test_verify.py:206
msgid "No run label"
msgstr "Kein Simulationsname"

#: test_verify.py:223
msgid "{} tests run, {} failed, {} errors"
msgstr "{} Tests ausgeführt, {} fehlgeschlagen, {} Fehler"

#: verification_state.py:34
msgid "No cash register with index {}."
msgstr "Keine Kassa mit Index {}."

#: verification_state.py:39
msgid "The last cash register has no registered start receipt."
msgstr "Die letzte Kassa hat keinen registrierten Startbeleg."

#: verification_state.py:57
msgid "Start Receipt"
msgstr "Startbeleg"

#: verification_state.py:58
msgid "Last Receipt"
msgstr "Letzter Beleg"

#: verification_state.py:59
msgid "Last Turnover Counter"
msgstr "Letzter Umsatzzähler"

#: verification_state.py:60
msgid "Need Restore Receipt"
msgstr "Benötigt Nullbeleg"

#: verification_state.py:165
msgid "Cash Register {}:"
msgstr "Kassa {}:"

#: verification_state.py:168
msgid "Used Receipt IDs"
msgstr "Benutzte Belegnummern"

#: verify.py:52
msgid "Malformed DEP"
msgstr "Ungültiges DEP"

#: verify.py:61
msgid "Malformed certificate: \"{}\""
msgstr "Ungültiges Zertifikat: \"{}\""

#: verify.py:70
msgid "Element \"{}\" missing from DEP"
msgstr "Element \"{}\" fehlt in DEP"

#: verify.py:80
msgid "GGS Cluster is not supported in an open system."
msgstr "GGS Verbund wird in offenen Systemen nicht unterstützt."

#: verify.py:101
msgid "Previous receipt is not \"{0}\"."
msgstr "Vorherige Rechnung ist nicht \"{0}\"."

#: verify.py:112
msgid "Receipt after restored signature system must not have any turnover."
msgstr ""
"Rechnung nach Wiederherstellung der Sicherheitseinrichtung darf keinen "
"Umsatz haben."

#: verify.py:122
msgid "Receipt ID already in use."
msgstr "Rechnungs-ID wird bereits verwendet."

#: verify.py:131
msgid "Turnover counter invalid."
msgstr "Umsatzzähler ungültig."

#: verify.py:140
msgid "Register ID changed."
msgstr "Kassen-ID geändert."

#: verify.py:150
msgid "Receipt was created before previous receipt."
msgstr "Rechnung wurde vor vorhergehender Rechnung erstellt."

#: verify.py:160
msgid "The system type changed."
msgstr "Typ des Systems wurde geändert."

#: verify.py:170
msgid "The size of the turnover counter changed."
msgstr "Die Länge des Umsatzzählers wurde geändert."

#: verify.py:179
msgid "No certificate specified in DEP and multiple groups used."
msgstr "Kein Zertifikat angegeben, aber DEP hat mehrere Gruppen."

#: verify.py:190
msgid "Certificate \"%s\" is not trusted."
msgstr "Zertifikat \"%s\" wird nicht vertraut."

#: verify.py:201
msgid "Certificate \"{}\" was not signed by \"{}\"."
msgstr "Zertifikat \"{}\" wurde nicht von \"{}\" signiert."

#: verify.py:212
msgid ""
"Two certificates with serial \"{0}\" detected (fingerprints \"{1}\" and "
"\"{2}\"). This may be an attempted attack."
msgstr ""
"Zwei Zertifikate mit Seriennummer \"{0}\" gefunden (Fingerabdrücke \"{1}\" "
"und \"{2}\"). Dies könnte ein versuchter Angriff sein."

#: verify.py:221
msgid "Initial receipt not signed."
msgstr "Startbeleg nicht signiert."

#: verify.py:229
msgid "Initial receipt has nonzero turnover."
msgstr "Startbeleg hat Umsatz ungleich Null."

#: verify.py:239
msgid "Initial receipt has not been chained to the cash register ID."
msgstr "Startbeleg ist nicht mit der Kassen-ID verkettet."

#: verify.py:249
msgid ""
"Initial receipt in cluster has not been chained to the previous cash "
"register's initial receipt."
msgstr ""
"Startbeleg in Verbund ist nicht mit dem Startbeleg der vorherigen Kassa "
"verkettet."

#: verify.py:258
msgid "Initial receipt is a dummy or reversal receipt."
msgstr "Startbeleg ist ein Trainings- oder Storno-Beleg."

#: verify.py:800
msgid "Verification successful."
msgstr "Überprüfung erfolgreich."

#: verify_receipt.py:41
msgid "Certificate serial mismatch."
msgstr "Zertifikat-Seriennummern stimmen nicht überein."

#: verify_receipt.py:48
msgid "Certificate serial invalid."
msgstr "Zertifikat-Seriennummer ungültig."

#: verify_receipt.py:56
msgid "No public key found."
msgstr "Kein öffentlicher Schlüssel gefunden."

#: verify_receipt.py:63
msgid "Invalid Signature."
msgstr "Ungültige Signatur."

#: verify_receipt.py:72
msgid "{} This is probably fine."
msgstr "{} Das ist wahrscheinlich in Ordnung."

#: verify_receipt.py:80
msgid "Signature System failed."
msgstr "Sicherheitseinrichtung ausgefallen."

#: verify_receipt.py:88
msgid "Invalid URL hash."
msgstr "Ungültige URL Prüfsumme."

#: verify_receipt.py:97
msgid "Invalid certificate provider."
msgstr "Ungültiger Zertifizierungsdiensteanbieter."

#: verify_receipt.py:105
msgid "Null receipt not signed."
msgstr "Nullbeleg nicht signiert."

#: verify_receipt.py:425
msgid "Line {: >3}: {}"
msgstr "Zeile {: >3}: {}"

#: verify_receipt.py:428
msgid "{} of {} receipts verified successfully."
msgstr "{} von {} Rechnungen wurden erfolgreich überprüft."
